components/ResultBanner.tsx

```diff
@@ -0,0 +1,119 @@
+import Image from "next/image";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import React from "react";
+import { useResponsive } from "../Hooks/useResponsive";
+
+type layoutPtops = {
+  children?: React.ReactNode;
+};
+
+export const BannerLayout: React.FC<layoutPtops> = (props) => {
+  const { children } = props;
+  const [isDesktop] = useResponsive();
+
+  return (
+    <section
+      className={`flex ${!isDesktop && "flex-col"} bg-[#161b22] p-4 rounded-xl justify-center items-center md:p-8`}
+    >
+      {children}
+    </section>
+  );
+};
+
+type imageProps = {
+  source: string;
```

Comment:
nitpick: can we use src to be consistent with the img tag?

Reply:
done